lib/spack/spack/cmd/mirror.py

```diff
@@ -224,6 +224,7 @@ def mirror_create(args):
         if error:
             tty.error("Failed downloads:")
             colify(s.cformat("$_$@") for s in error)
+            tty.die("Some downloads failed.")
```

Comment:
I think the text is superfluous since we already write on error, but I don't know how to return a non-zero error code without writing at least a `""` in `tty.die("")` which then looks even more odd :)

Reply:
```python
import sys

sys.exit(1)
```

Reply:
ah yes, updated!